### PR TITLE
make native-tls optional

### PR DESCRIPTION
**Pull request by AdrianCX**

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -85,10 +88,15 @@ impl SdkmsClientBuilder {
...
-                Arc::new(client)
+                #[cfg(feature = "native-tls")]
+                {
+                    let ssl = NativeTlsClient::new()?;
+                    let connector = HttpsConnector::new(ssl);
+                    let client = HyperClient::with_connector(Pool::with_connector(Default::default(), connector));
+                    Arc::new(client)
+                }
+                #[cfg(not(feature = "native-tls"))]
+                panic!("Unsuported");
```
Comment (mzohreva) [inline src/client.rs]: This should be more descriptive, e.g. `You should either provide a hyper Client or compile this crate with hyper-native-tls feature`

File: src/lib.rs [hunk 2]
```diff
@@ -67,6 +67,8 @@ extern crate uuid;
 extern crate bitflags;
 
 extern crate hyper;
+
+#[cfg(feature = "native-tls")]
```
Comment (mzohreva) [inline src/lib.rs]: These cfg attributes should be changed to the following:
```rust
#[cfg(feature = "hyper-native-tls")]
```
I'm surprised that this even compiles!

File: Cargo.toml [hunk 3]
```diff
@@ -12,14 +12,17 @@ documentation = "https://docs.rs/sdkms"
...
 [dependencies]
 chrono = "0.4"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
-hyper-native-tls = "0.3"
+hyper-native-tls = { version = "0.3", optional = true }
 hyper = "0.10"
-uuid = { version = "0.7", features = ["serde", "v4"] }
+uuid = { version = "0.6.3", features = ["v4", "serde"] }
```
Comment (mzohreva) [inline Cargo.toml]: This seems unrelated

Comment (AdrianCX) [reply Cargo.toml]: removed, it was unrelated

File: src/lib.rs (hunk 2 above)
Comment (AdrianCX) [reply src/lib.rs]: switched to hyper-native-tls, thanks

File: src/client.rs (hunk 1 above)
Comment (AdrianCX) [reply src/client.rs]: Changed to the text you provided, thanks

**Comment by mzohreva** [comment]
Looks good. Please bump the version in Cargo.toml to `0.2.1` so we can publish after merge.
